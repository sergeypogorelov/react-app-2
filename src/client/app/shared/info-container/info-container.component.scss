@import '../../../styles/variables';

.info-container {
  height: 55px;
  padding: 0 120px;
  background-color: $c-background-alt;
}
