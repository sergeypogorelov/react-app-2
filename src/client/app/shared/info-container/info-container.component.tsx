import './info-container.component.scss';

import React, { FunctionComponent } from 'react';

export const InfoContainer: FunctionComponent<{}> = (props) => {
  return (
    <div className="info-container d-flex align-items-center justify-content-between">
      {props.children}
    </div>
  );
};
