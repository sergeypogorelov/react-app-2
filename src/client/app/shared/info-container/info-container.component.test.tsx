import React from 'react';
import renderer from 'react-test-renderer';

import { InfoContainer } from './info-container.component';

test('InfoContainer component should render correctly', () => {
  const tree = renderer.create(<InfoContainer>test</InfoContainer>).toJSON();
  expect(tree).toMatchSnapshot();
});
