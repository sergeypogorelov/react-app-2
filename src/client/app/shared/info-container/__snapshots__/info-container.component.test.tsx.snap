// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`InfoContainer component should render correctly 1`] = `
<div
  className="info-container d-flex align-items-center justify-content-between"
>
  test
</div>
`;
